qiita_pet/handlers/study_handlers/sample_template.py

```diff
@@ -277,9 +283,19 @@ def post(self):
         study_id = int(self.get_argument('study_id'))
         filepath = self.get_argument('filepath')
         data_type = self.get_argument('data_type')
+        direct_upload = self.get_argument('data_type', False)
+
+        if direct_upload:
+            direct_upload = True
```

Comment:
Is `directd_upload` a boolean value? This seems odd since you check this value and then assign it to True. Would you mind explaining what's going on here?

Reply:
direct_upload is a b'true' so just making sure it's a boolean; changing for clarity.